CONTRIBUTING.md

```diff
@@ -0,0 +1,144 @@
+# Contributing to SKY UX
+
+We encourage contributions from all SKY UX users. To contribute, follow this step-by-step guide. For more information about our contribution guidelines, see [SKY UX Contribution Guidelines](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines).
+
+- [Contribution Process](#contribution-process)
+- [Coding Rules](#coding-rules)
+- [Commit Message Guidelines](#commit-message-guidelines)
+- [Local Development Cheatsheet](#local-development)
+
+## <a name="contribution-process"></a> Contribution Process
```

Comment:
```suggestion
## <a name="contribution-process"></a> Contribution process
```
Potentially dumb question: In these H2s, should the header text be _inside_ the `a` tags instead of after the closing tag?

Reply:
Definitely a mistake. Good catch!